Dockerfile.nostack

```diff
@@ -0,0 +1,24 @@
+ARG STACK_VERSION=lts-13
+FROM fpco/stack-build:$STACK_VERSION as builder
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y clang \
+    && git clone https://github.com/carp-lang/carp . \
+    && stack build && stack install && stack clean \
+    && mv /root/.local/bin /app/bin \
+    && rm -rf /app/.stack-work/install
+
+
+FROM ubuntu:16.04
+
+COPY --from=builder /app /opt/carp
+
+ENV CARP_DIR=/opt/carp/
+ENV PATH=$PATH:"${CARP_DIR}bin"
+
+RUN apt-get update && apt-get install -y clang libgmp10
```

Comment:
Why do we need `libgmp`?

Reply:
@hellerve  carp won't start without it :thinking:

Reply:
Weird; why is it not required by the other image? Is it pulled from some lower layer there?

Reply:
@hellerve  most likely stack image has it. :thinking:  You can test it by running the image and searching for the lib.

Reply:
I was able to verify that this is a problem; superficial googling seems to suggest that this is because Haskell depends on it and links to it dynamically. That makes sense, I guess! :)